### ISSUE TITLE
CKV_GCP_79: Update latest Major Postgres version to 16

### ISSUE THREAD

**Issue by lucaspierru-convelio** [bridgecrewio/checkov#6689]
**Describe the issue**
CKV_GCP_79: "Ensure SQL database is using latest Major version" is outdated now that Postgres 16 is available on GCP. We should update this rule to ensure it checks for Postgres 16 instead of 15.

**Examples**
This triggers check CKV_GCP_79:
```terraform
resource "google_sql_database_instance" "default" {
  database_version = "POSTGRES_16"
```
when it should pass the check.

https://github.com/bridgecrewio/checkov/blob/91932f8c23eae0eaf9fd2b705a936c06c45185c5/checkov/terraform/checks/resource/gcp/CloudSqlMajorVersion.py#L17 should be updated to `"POSTGRES_16"`

**Version (please complete the following information):**
 - Checkov Docker image version 3.2.238

**Comment by bo156** [comment]
@lucaspierru-convelio thanks for the comment, would you want to open a PR to add this behavior?

**Comment by lucaspierru-convelio** [comment]
@bo156 @qdelettre took care of it, its PR is still open and needs reviewing: https://github.com/bridgecrewio/checkov/pull/6695
